package.json

```diff
@@ -73,8 +73,8 @@
     "azure-arm-website": "0.11.2",
     "azure-arm-datalake-analytics": "0.4.3",
     "azure-arm-datalake-store": "0.4.2",
-    "azure-arm-batch": "0.2.1",
-    "azure-batch": "0.5.1",
+    "azure-arm-batch": "0.4.0",
```

Comment:
Well, looks like the CI job failed because it couldn't find the version specified here, so I'm going to revert my change to this file.